pinot-spi/src/main/java/org/apache/pinot/spi/utils/QueryOptionsUtils.java

```diff
@@ -16,9 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.core.util;
+package org.apache.pinot.spi.utils;
```

Comment:
do we need to move this Util to SPI? looks like the only place it is being used is common. cant we move it into common?